packages/smart/test/turbo-test.ts

```diff
@@ -221,6 +224,17 @@ describe("Turbo", () => {
     expect(await none.balanceOf(signer.address)).to.equal(0);
   });
 
+  it("cannot mint sets after market resolution", async () => {
+    const setsToMint = shareFactor.mul(100);
+    const costToMint = setsToMint.div(shareFactor);
+
+    await collateral.faucet(costToMint);
+    await collateral.approve(marketFactory.address, costToMint);
+    await expect(marketFactory.mintShares(marketId, setsToMint, signer.address)).to.be.rejectedWith(
+      "VM Exception while processing transaction: revert Cannot mint shares for resolved market"
+    );
+  });
+
   it("can create a test balancer pool", async () => {
```

Comment:
Just to make double sure, need tests to verify user can sell and LPs can remove liquidity after market is resolved.

Reply:
We don't want users to be able to sell since after resolution they only claim.

I will add a test for removing liquidity.

Reply:
Caught a bug!